### ISSUE TITLE
answerTagsSearch highlights specific tag

### ISSUE THREAD

**Issue by wgrundlingh** [soscripted/sox#268]
### Current Behaviour

A specific tag is highlighted when adding the tag to an *answer search*. Below, the `{debugging}` tag is highlighted on [TeX.SE](http://tex.stackexchange.com):

![image](https://cloud.githubusercontent.com/assets/1776164/23979769/cbe83034-09b8-11e7-825e-fc78836b079d.png)

This is what it looks like on [SO](http://stackoverflow.com):

![image](https://cloud.githubusercontent.com/assets/1776164/23979825/283660b8-09b9-11e7-8253-d2253f1fdf67.png)

Not sure whether this applies to other tags as well.

---

### SOX Errors logged to the browser console -- F12 (if available)

```
SOX:  undefined
VM4669:565 SOX:  Object
VM4669:565 SOX:  DEBUGGING SOX VERSION 2.0.24
VM4669:565 SOX:  ----------------saved variables---------------------
VM4669:589 SOX:  logging sox stored values --- 
VM4669:589 SOX:  access token set
VM4669:589 SOX:  SOX-debug
VM4669:589 SOX:  true
VM4669:589 SOX:  SOXSETTINGS
VM4669:589 SOX:  ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-fixedTopbar","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-scrollToTop","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-enhancedEditor","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Chat-chatEasyAccess","Chat-renameChat","Voting-betterCSS","Voting-grayOutVotes","Voting-stickyVoteButtons","Voting-disableOwnPostVoteButtons","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-showMetaReviewCount","Extras-copyCode","Extras-dailyReviewBar"]
VM4669:589 SOX:  downvotedPostsEditAlert
VM4669:589 SOX:  {"347192":{"questionId":"347192","addedDate":1483602847263,"sitename":"tex","lastCheckedTime":1484368203293},"347197":{"questionId":"347192","addedDate":1483602833281,"sitename":"tex","lastCheckedTime":1484368203396},"347487":{"questionId":"347484","addedDate":1483740886378,"sitename":"tex","lastCheckedTime":1484368203507}}
VM4669:589 SOX:  editReasons
VM4669:589 SOX:  [["updated markdown","updated markdown"],["retag","retag"]]
VM4669:589 SOX:  metaNewQuestionAlert-lastQuestions
VM4669:589 SOX:  {"meta.tex":"How to including multimedia files in beamer [Work in progress, please ignor for the moment]"}
VM4669:565 SOX:  ----------------end saved variables---------------------
VM4669:565 SOX:  initializing SOX dialog
VM4669:565 SOX:  injecting features into dialog
VM4669:565 SOX:  Object
VM4669:565 SOX:  running addAuthorNameToInboxNotifications
VM4669:565 SOX:  observe: .inbox-dialog
VM4669:565 SOX:  Object
VM4669:565 SOX:  running answerTagsSearch
VM4669:565 SOX:  SOX Access Token: SET
VM4669:565 SOX:  Getting From API with URL: https://api.stackexchange.com/2.2/questions/208881;110760;71384;147077;1915…&site=tex&key=lL1S1jr2m*DRwOvXMPp26g((&access_token=e2VVaHLYkmc2GEGYEiIyNQ))
VM4669:565 SOX:  SOX Access Token: SET
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  running fixedTopbar
VM4669:565 SOX:  observe: #notify-container,#notify--1
VM4669:565 SOX:  Object
VM4669:565 SOX:  running highlightQuestions
VM4669:565 SOX:  Object
VM4669:565 SOX:  running isQuestionHot
VM4669:565 SOX:  Object
VM4669:565 SOX:  running markEmployees
VM4669:565 SOX:  SOX Access Token: SET
VM4669:565 SOX:  Object
VM4669:565 SOX:  running metaChatBlogStackExchangeButton
VM4669:565 SOX:  Object
VM4669:565 SOX:  running scrollToTop
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  running standOutDupeCloseMigrated
VM4669:565 SOX:  observe: #user-tab-questions, #question-mini-list
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  running addSBSBtn
VM4669:565 SOX:  observe: .wmd-preview.sbs-on
VM4669:565 SOX:  Object
VM4669:565 SOX:  running editComment
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  Object
VM4669:565 SOX:  running flagOutcomeTime
VM4669:565 SOX:  Object
(unknown) SOX:  running flagPercentages
(unknown) SOX:  Object
(unknown) SOX:  running flagPercentageBar
(unknown) SOX:  Object
(unknown) SOX:  running hideHireMe
(unknown) SOX:  Object
(unknown) SOX:  running hideLoveThisSite
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  Object
(unknown) SOX:  observe: .new_comment
(unknown) SOX:  observe: li[id^="wmd-redo-button"], textarea[id^="wmd-input"]
(unknown) SOX:  observe: .reviewable-post, .review-content
```

---

### Steps to reproduce

The following search query on [TeX.SE](http://tex.stackexchange.com) identifies the behaviour:
  http://tex.stackexchange.com/search?q=is%3Aanswer+%5Bdebugging%5D
Here's the same on [SO](http://stackoverflow.com):
  http://stackoverflow.com/search?q=is%3Aanswer+%5Bdebugging%5D

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-fixedTopbar","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-scrollToTop","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-enhancedEditor","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Chat-chatEasyAccess","Chat-renameChat","Voting-betterCSS","Voting-grayOutVotes","Voting-stickyVoteButtons","Voting-disableOwnPostVoteButtons","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-showMetaReviewCount","Extras-copyCode","Extras-dailyReviewBar"]

**Comment by shu8** [comment]
@wgrundlingh thanks for the bug report; we've both been really busy recently, but we'll look into this soon! :0

**Comment by wgrundlingh** [comment]
Hey @shu8, no worries. Your time is appreciated.

**Comment by shu8** [comment]
@wgrundlingh so, this was because I hadn't put a word boundary around the check that makes meta mod-only tags a different colour (bug, discussion, feature-request, support), and so it highlighted 'debugging' because it has the word 'bug' in it! 😆 

Fixed in [dev 2.0.25](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)

Thanks for bearing with me! :)
